index.html

```diff
@@ -66,5 +72,236 @@ <h3>
+                </li>
+                <li class="C Li">
+                  <i class="bi bi-circle-fill icon"></i> 2019
+                </li>
+              </ul>
+            </div>
+
+            <div class="box2">
+              <p class="card-para">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="box2-ul">
+                <li class="box2-li">HTML</li>
+                <li class="box2-li">Css</li>
+                <li class="box2-li">JavaScript</li>
+              </ul>
+              <button class="big-btn">See project</button>
+            </div>
+          </block>
+        </cards>
+
+        <cards class="cards">
+          <block class="block1">
+            <img class="card-img" src="/images/Snapshoot.png" alt="" />
+          </block>
+
+          <block class="block2">
+            <div class="box1">
+              <h1 class="title1">Gateway</h1>
+              <ul class="frame1">
+                <li class="A Li">
+                  <i class="bi bi-circle-fill icon"></i> CANOPY
+                </li>
+                <li class="B Li">
+                  <i class="bi bi-circle-fill icon"></i> Back End dev
+                </li>
+                <li class="C Li">
+                  <i class="bi bi-circle-fill icon"></i> 2019
+                </li>
+              </ul>
+            </div>
+
+            <div class="box2">
+              <p class="card-para">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="box2-ul">
+                <li class="box2-li">HTML</li>
+                <li class="box2-li">Css</li>
+                <li class="box2-li">JavaScript</li>
+              </ul>
+              <button class="big-btn">See project</button>
+            </div>
+          </block>
+        </cards>
+      </works>
+    </div>
+
+    <!-- about myself -->
+
+    <About class="about">
+      <section class="section1">
+        <h1 class="primary-text">About Myself</h1>
+        <p class="paragraph about-para">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <a href="#" class="connect about-connect">Let's connect</a>
+        <ul class="social">
+          <li>
+            <a href="https://twitter.com/ankit26k">
+              <img class="icon" src="/images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+
+          <li>
+            <a href="https://www.linkedin.com/in/ankit26k/">
+              <img class="icon" src="/images/Linkedin.png" alt="linkedin"
+            /></a>
+          </li>
```

Comment:
Kindly check again the path to all your images as any of them shows up in the server 

![temp1 student images missing](https://user-images.githubusercontent.com/59707859/227724280-b2eb7df5-012e-483c-a863-86e8430f32cb.PNG)